LayoutTests/platform/mac/TestExpectations

```diff
@@ -2456,7 +2456,7 @@ http/tests/webgpu/webgpu/api/validation/image_copy/buffer_related.html [ Skip ]
 http/tests/webgpu/webgpu/api/validation/image_copy/buffer_texture_copies.html [ Skip ]
 http/tests/webgpu/webgpu/api/validation/image_copy/texture_related.html [ Skip ]
 http/tests/webgpu/webgpu/api/validation/image_copy/layout_related.html [ Skip ]
-http/tests/webgpu/webgpu/api/validation/createSampler.html [ Pass ]
```

Comment:
this was observed to be a flaky failure (1-3 failures out of 100 iterations), disabling until it is fixed